CHANGES.md

```diff
@@ -103,6 +104,7 @@
 
 * Pipeline dependencies supplied through `--requirements_file` will now be staged to the runner using binary distributions (wheels) of the PyPI packages for linux_x86_64 platform ([BEAM-4032](https://issues.apache.org/jira/browse/BEAM-4032)). To restore the behavior to use source distributions, set pipeline option `--requirements_cache_only_sources`. To skip staging the packages at submission time, set pipeline option `--requirements_cache=skip` (Python).
 * The Flink runner now supports Flink 1.14.x ([BEAM-13106](https://issues.apache.org/jira/browse/BEAM-13106)).
+* The Interactive runner now supports remotely executing pipelines on Dataproc when the underlying runner is the Flink runner (Python) ([BEAM-14071](https://issues.apache.org/jira/browse/BEAM-14071)).
```

Comment:
This can be shortened to: 
"Interactive Beam now supports remotely executing Flink pipelines on Dataproc"

Reply:
That's better, thanks!